packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -202,6 +202,18 @@ export default abstract class BaseCall implements IWebRTCCall {
     }
   }
 
+  /**
+   * Hangs up the call.
+   *
+   * @param params _For internal use_ Specify custom hangup cause and call ID
```

Comment:
It doesn't appear you can hide a parameter or mark it for internal use in Typedoc, so I've added a note in the description.